.babelrc

```diff
@@ -11,19 +10,28 @@
         },
         "useBuiltIns": "usage"
       }
-    ]
+    ],
+    "@babel/preset-typescript"
   ],
   "plugins": [
-    "styled-components",
+    "babel-plugin-transform-typescript-metadata",
     [
       "@babel/plugin-proposal-decorators",
       {
         "legacy": true
       }
     ],
-    "@babel/plugin-transform-destructuring",
-    "@babel/plugin-transform-regenerator",
-    "transform-class-properties"
+    "@babel/plugin-proposal-class-properties",
+    [
+      "transform-inline-environment-variables",
+      {
+        "include": [
+          "SOURCE_COMMIT",
+          "SOURCE_VERSION"
```

Comment:
I don't see these env vars in codebase.

Reply:
They come from the deployment process

Reply:
Yeah, my bad, had case-sensitivity toggled while searching 🙈